### ISSUE TITLE
java.lang.reflect.InvocationTargetException

### ISSUE THREAD

**Issue by scruel** [MuntashirAkon/AppManager#1387]
### Please check before submitting an issue

- [X] I know what my device, OS and App Manager versions are
- [X] I know how to take logs
- [X] I know how to reproduce the issue which may not be specific to my device

### Describe the bug

```
05-31 01:23:09.014  6331  8529 E SuggestManager: openApp name = io.github.muntashirakon.AppManager
05-31 01:23:09.015  6331 10617 E AppPredict: insert result = 51516
05-31 01:23:09.021  8343  8708 E WakePathManager: caller not found!
05-31 01:23:09.021  8343  8708 E WakePathManager: java.lang.reflect.InvocationTargetException
05-31 01:23:09.021  8343  8708 E WakePathManager:       at java.lang.reflect.Method.invoke(Native Method)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at com.miui.permission.support.util.ReflectUtil.callObjectMethod(Unknown Source:8)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at com.miui.authmanager.wakepath.WakePathManager.sendWakePathPermissionRecord(Unknown Source:55)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at com.miui.authmanager.wakepath.WakePathManager.access$500(Unknown Source:0)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at com.miui.authmanager.wakepath.WakePathManager$1.handleMessage(Unknown Source:29)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at android.os.Handler.dispatchMessage(Handler.java:106)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at android.os.Looper.loopOnce(Looper.java:211)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at android.os.Looper.loop(Looper.java:300)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at android.os.HandlerThread.run(HandlerThread.java:67)
05-31 01:23:09.021  8343  8708 E WakePathManager: Caused by: android.content.pm.PackageManager$NameNotFoundException: io.github.muntashirakon.AppManager
05-31 01:23:09.021  8343  8708 E WakePathManager:       at android.app.ApplicationPackageManager.getPackageInfoAsUser(ApplicationPackageManager.java:254)
05-31 01:23:09.021  8343  8708 E WakePathManager:       at android.app.ApplicationPackageManager.getPackageInfoAsUser(ApplicationPackageManager.java:242)
05-31 01:23:09.021  8343  8708 E WakePathManager:       ... 9 more
05-31 01:23:09.021  2145  2884 E CellBroadcastUtils: getDefaultCellBroadcastReceiverPackageName: no package found
05-31 01:23:09.048  1231  1338 E ANDR-PERF-LM: MetaMeter: handleAsync() 166: AdaptLaunch Invalid Async
05-31 01:23:09.052  2145  2959 E MiuiActivityController: MiuiLog-ActivityObserver: There was something wrong : null
05-31 01:23:09.054  6331  8529 E SuggestManager: openApp name = com.lbe.security.miui
05-31 01:23:09.056  6331 10617 E AppPredict: insert result = 51517
05-31 01:23:09.060 14978 14978 E ecurity.miui:ui: Invalid ID 0x40000000.
05-31 01:23:09.060 14978 14978 E ecurity.miui:ui: Invalid ID 0x0000001c.
05-31 01:23:09.082  2145  2959 E MiuiActivityController: MiuiLog-ActivityObserver: There was something wrong : null
05-31 01:23:09.083  8343  8513 E AppOpsCompat: reflect checkOpNoThrow exception!
05-31 01:23:09.083  8343  8513 E AppOpsCompat: java.lang.reflect.InvocationTargetException
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at java.lang.reflect.Method.invoke(Native Method)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at com.miui.permission.support.util.ReflectUtil.callObjectMethod(Unknown Source:8)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at android.app.AppOpsManagerCompat.checkOpNoThrow(Unknown Source:34)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at com.miui.privacy.autostart.AutoRevokePermissionManager.isASPermissionAutoRevoke(Unknown Source:12)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at com.miui.privacy.autostart.AutoRevokePermissionManager.access$200(Unknown Source:0)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at com.miui.privacy.autostart.AutoRevokePermissionManager$1.onForegroundInfoChanged(Unknown Source:20)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at miui.process.IForegroundInfoListener$Stub.onTransact(IForegroundInfoListener.java:66)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at android.os.Binder.execTransactInternal(Binder.java:1290)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at android.os.Binder.execTransact(Binder.java:1249)
05-31 01:23:09.083  8343  8513 E AppOpsCompat: Caused by: java.lang.IllegalArgumentException: io.github.muntashirakon.AppManager not found from 10200
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at android.os.Parcel.createExceptionOrNull(Parcel.java:3015)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at android.os.Parcel.createException(Parcel.java:2995)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at android.os.Parcel.readException(Parcel.java:2978)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at android.os.Parcel.readException(Parcel.java:2920)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at com.android.internal.app.IAppOpsService$Stub$Proxy.checkOperation(IAppOpsService.java:1199)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  at android.app.AppOpsManager.checkOpNoThrow(AppOpsManager.java:8999)
05-31 01:23:09.083  8343  8513 E AppOpsCompat:  ... 9 more
05-31 01:23:09.085  6331  8529 E SuggestManager: openApp name = io.github.muntashirakon.AppManager
05-31 01:23:09.088  6331 10617 E AppPredict: insert result = 51518
05-31 01:23:09.088  8343  8708 E WakePathManager: caller not found!
05-31 01:23:09.088  8343  8708 E WakePathManager: java.lang.reflect.InvocationTargetException
05-31 01:23:09.088  8343  8708 E WakePathManager:       at java.lang.reflect.Method.invoke(Native Method)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at com.miui.permission.support.util.ReflectUtil.callObjectMethod(Unknown Source:8)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at com.miui.authmanager.wakepath.WakePathManager.sendWakePathPermissionRecord(Unknown Source:55)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at com.miui.authmanager.wakepath.WakePathManager.access$500(Unknown Source:0)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at com.miui.authmanager.wakepath.WakePathManager$1.handleMessage(Unknown Source:29)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at android.os.Handler.dispatchMessage(Handler.java:106)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at android.os.Looper.loopOnce(Looper.java:211)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at android.os.Looper.loop(Looper.java:300)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at android.os.HandlerThread.run(HandlerThread.java:67)
05-31 01:23:09.088  8343  8708 E WakePathManager: Caused by: android.content.pm.PackageManager$NameNotFoundException: io.github.muntashirakon.AppManager
05-31 01:23:09.088  8343  8708 E WakePathManager:       at android.app.ApplicationPackageManager.getPackageInfoAsUser(ApplicationPackageManager.java:254)
05-31 01:23:09.088  8343  8708 E WakePathManager:       at android.app.ApplicationPackageManager.getPackageInfoAsUser(ApplicationPackageManager.java:242)
05-31 01:23:09.088  8343  8708 E WakePathManager:       ... 9 more
05-31 01:23:09.088  2145  2884 E CellBroadcastUtils: getDefaultCellBroadcastReceiverPackageName: no package found
05-31 01:23:09.109  1231  1338 E ANDR-PERF-LM: MetaMeter: handleAsync() 166: AdaptLaunch Invalid Async
05-31 01:23:09.115  2145  2959 E MiuiActivityController: MiuiLog-ActivityObserver: There was something wrong : null
05-31 01:23:09.116  6331  8529 E SuggestManager: openApp name = com.lbe.security.miui
```

### To Reproduce

- Open AppManager
- logcat *:E

### Device info

- Device: Xiaomi cas
- OS Version: Android 13
- App Manager Version: 3.1.6
- Mode: NonRoot

**Comment by MuntashirAkon** [comment]
These logs do not seem to be anything to do with App Manager. You need to filter App Manager's logs using its PID.

**Comment by MuntashirAkon** [comment]
Closed via #1388
